components/director/internal/graphql/scalars/annotations_test.go

```diff
@@ -0,0 +1,51 @@
+package scalars
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAnnotations_UnmarshalGQL(t *testing.T) {
+	//given
+	a := Annotations{}
+	fixAnnotations := map[string]interface{}{
+		"annotation1": "val1",
+	}
+	expectedAnnotations := Annotations{
+		"annotation1": "val1",
+	}
+
+	//when
+	err := a.UnmarshalGQL(fixAnnotations)
+
+	//then
+	require.NoError(t, err)
+	assert.Equal(t, a, expectedAnnotations)
+}
+
+func TestAnnotations_MarshalGQL(t *testing.T) {
```

Comment:
This smells like two test cases. Consider splitting it.

Reply:
done